manifests/4.7/cluster-logging.v4.7.0.clusterserviceversion.yaml

```diff
@@ -159,9 +159,9 @@ spec:
   - type: SingleNamespace
     supported: true
   - type: MultiNamespace
-    supported: true
+    supported: false
   - type: AllNamespaces
-    supported: true
+    supported: false
```

Comment:
@vimalk78  Are we dropping support of `MultipleNamespace` as well as `AllNamespaces`?

Reply:
We actually never supported this one, right? cluster-logging-operator was always capable to handle requests in `openshift-logging`.